modules/display.c

```diff
@@ -11439,22 +11439,36 @@ static void mdy_brightness_init(void)
 {
     gulong tmp = 0;
 
-    /* If possible, obtain maximum brightness level */
-    if( !mdy_brightness_level_maximum_path ) {
-        mce_log(LL_NOTICE, "No path for maximum brightness file; "
-                "defaulting to %d",
-                mdy_brightness_level_maximum);
+    /* Check if max_brightness override has been configured */
+    if( mce_conf_has_key(MCE_CONF_DISPLAY_GROUP,
```

Comment:
mce_conf_has_key check seems to be redundant

Reply:
Getter assumes that a missing key is something unexpected and complains -> to avoid journal spamming pre-check is used for values that are meant to be optional.

Reply:
Makes sense then!